src/app/state/rootInitialState.ts

```diff
@@ -9,7 +9,7 @@ import messagesInitialState, {
 } from '../../messages/state/messagesInitialState';
 import toolsInitialState, {
   ToolsState,
-} from '../../tools/state/toolsInitialState';
+} from '../../tools/types/toolsInitialState';
```

Comment:
A minor quibble: `toolsInitialState` is not strictly state though it is stored in the `types` directory.

Reply:
Yes, looking at this specific file it's quite obvious this is not following the rest of the app's structure. I'll change that now.